compiler/hash-lower/src/discover.rs

```diff
@@ -144,6 +137,10 @@ impl FnDiscoverer<'_> {
                     ModMemberValue::Fn(_) => {
                         // We've already found this one.
                     }
+                    ModMemberValue::Intrinsic(_) => {
+                        // @@Todo: we should probably add all the intrinsic
+                        // functions?
+                    }
```

Comment:
It shouldn't really need to do anything here, after TC, it is up to the backend to emit code for the intrinsic. For example take a look at what the backends do for the various intrinsics:
https://github.com/hash-org/hashc/blob/b17713d5deb9ce802fdaf8f527dcf06d27c56c3b/compiler/hash-codegen/src/lower/intrinsics.rs#L31-L68